ddtrace/internal/periodic.py

```diff
@@ -184,8 +184,12 @@ def interval(self, value):
         if self._worker:
             self._worker.interval = value
 
-    def _start(self):
-        # type: () -> None
+    def _start(
+        self,
+        *args,  # type: typing.Any
```

Comment:
Is it not possible to have a more precise signature for these methods?

Reply:
Not really as it's just passing the plate to the inheriting class.
And since the parent classes are accepting any arguments for `start` to allow inheriting class to get those, it makes it impossible to set a more specific type there.